app/preview/TemplateParams.scala

```diff
@@ -741,6 +741,36 @@ object TemplateParams {
       "receivedDate" -> "20 April 2017", "identifier" -> "12345",
       "subject" -> "subject"
     ),
+    "transactionEngineHMRCPSAPRACAUTHSuccess" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject"
+    ),
+    "transactionEngineHMRCPSAPRACAUTHFailure" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject", "errorType" -> "business",
+      "errorTextBlock" -> "<p>Error Code: 123 Error Message: An Error</p>"
+    ),
+    "transactionEngineHMRCPSAPRACAUTHFailure" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject",
+      "errorType" -> "fatal",
+      "errorNumber" -> "123", "errorMessage" -> "This is an error"
+    ),
+    "transactionEngineHMRCPSAPRACDEAUTHSuccess" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject"
+    ),
+    "transactionEngineHMRCPSAPRACDEAUTHFailure" -> Map(
```

Comment:
transactionEngineHMRCPSAPRACDEAUTHFailure is added twice. Could you correct it, please?